Sprint-1/interpret/to-pounds.js

```diff
@@ -24,4 +29,16 @@ console.log(`£${pounds}.${pence}`);
 // Try and describe the purpose / rationale behind each step
 
 // To begin, we can start with
-// 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 1. const penceString = "399p": initializes a string variable with the value "399p"
+// 3. initializes a variable that removes the "p" at the end, it uses method .substring
+// which returns the part of the string corresponding to the index and end index
+//-1 excluding the p
+//9. uses the method padstart to add "0" to the start of string penceStringWithoutTrailingP,
+// it returns the same string unchanged because the target length is equal to the current length.
+// this effectively multiplies any value input as penceStringWithoutTrailingP by 100
+//12. uses the substring method to return the first value of the string, this is to get
+// the pound. it also used length -2 to make sure its in a valid range so the program
+// can go through the steps with any value input.
+//18.the code gets the last two digits from the paddedPenceNumberString and ensures that the result
```

Comment:
Can we expect this program to work as intended even if we deleted `.padEnd(2, "0")` from the code?

Reply:
no, it will not work for single digit or double digits input

Reply:
Well, can you find a value of `penceString` to show that the function won't work properly without `.padEnd(2, "0")`?